packages/forms-web-app/src/sass/expand-collapse-comments.scss

```diff
@@ -0,0 +1,12 @@
+.toggle-button {
+    display: none;
+}
+
+.js-enabled .toggle-button {
+    display: inline;
```

Comment:
I did try using a `<noscript>` in the nunjucks to hide buttons if JS wasnt enabled instead of doing this but couldnt get it to work. The buttons were keen to stay, this way removes them if the default full text is shown and no JS enabled

Reply:
So BO seem to have something similar:
https://github.com/Planning-Inspectorate/appeals-back-office/blob/main/appeals/web/src/client/components/excerpts/excerpt.js

Havne't gone over it myself yet, might be worth seeing if there's bits in there we can reuse, e.g. tests + fallbacks